src/app/modules/item/components/task-loader/task-loader.component.ts

```diff
@@ -1,14 +1,31 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Subscription, timer } from 'rxjs';
+import { SECONDS } from 'src/app/shared/helpers/duration';
 
 @Component({
   selector: 'alg-task-loader',
   templateUrl: './task-loader.component.html',
   styleUrls: [ './task-loader.component.scss' ]
 })
-export class TaskLoaderComponent {
+export class TaskLoaderComponent implements OnInit, OnDestroy {
 
   @Input() label = '';
+  @Input() delayedLabel = '';
+  @Input() delay = 5;
+
+  showDelayedLabel = false;
+  subscription?: Subscription;
 
   constructor() { }
 
+  ngOnInit(): void {
+    this.subscription = timer(this.delay*SECONDS).subscribe(() => {
```

Comment:
maybe only if `this.delayedLabel` is set  (if so, this should be optional instead of defaulting to empty string)

Reply:
Fixed